app/src/main/java/org/mozilla/fenix/components/metrics/Event.kt

```diff
@@ -31,5 +31,10 @@ sealed class Event {
          * Event recording the first time a URI is loaded in Firefox in a 24 hour period.
          */
         object FirstUriLoadForDay : GrowthData("ja86ek")
+
+        /**
+         * Event recording the first time Firefox is used 3 days in a row in the first week of install.
+         */
+        object FirstWeekSeriesActivity : GrowthData("20ay7u")
```

Comment:
are the strings codes from the Adjust SDK?

Reply:
Yes. These codes are generated per event/project combo which means they need to be different during testing. If we continue down this path we should invest in a dynamic system for inserting them at build time or something similar